spec/src/main/asciidoc/overview.adoc

```diff
@@ -27,17 +27,18 @@ https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-read
 
 In this scenario health checks are used to determine if a computing node needs to be discarded (terminated, shutdown) and eventually replaced by another (healthy) instance.
 
-The MP-HC architecture consists of a single `/health` endpoint in a MicroProfile runtime that represents the status of
-the entire runtime. This `/health` endpoint is expected to be associated with a configurable context, such as a web
-application deployment, that can be configured with settings such as port, virtual-host, security, etc.
+The MP-HC architecture consists of two `/health/ready` and `/health/live` endpoints in a MicroProfile runtime that respectively represents the readiness and the liveness of the entire runtime.
+For backward compatibility, a 3rd endpoint `/health` can also be used to provide a custom Health Check.
+These endpoints are expected to be associated with a configurable context, such as a web application deployment, that can be configured with settings such as port, virtual-host, security, etc.
 Further, the MP-HC defines the notion of a procedure that represents the health of a particular
-subcomponent of an application. There can be zero or more procedures in an application, and the
-overall health of the application as reflected via the `/health` endpoint is the logical AND of
-all of the procedure health status.
+subcomponent of an application.
 
-The 1.0 version of the MP-HC specification does not define how the `/health` endpoint may be partitioned in the event
+In an application, there can be zero or more procedures bound to a given health endpoint.
+The overall application health for a given endpoint is the logical AND of all of the procedure bound to it.
+
+The current version of the MP-HC specification does not define how the `/health/live` endpoint may be partitioned in the event
 that the MicroProfile runtime supports deployment of multiple applications. If an implementation wishes to
-support multiple applications within a MicroProfile runtime, the semantics of the `/health` endpoint are
+support multiple applications within a MicroProfile runtime, the semantics of the `/health/live` and `/health/ready` endpoints are
```

Comment:
maybe we should say here rather "individual endpoints". Even for the /health endpoint this property should be preserved.